scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -163,22 +148,19 @@ ssh-keyscan -H localhost >> ~/.ssh/known_hosts 2>/dev/null
 if [[ ${CI:=} == "true" && ${RUN_TESTS} == "true" ]] ; then
     echo
     echo " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-    echo "  Setting default parallellism to 2 because we can run out of memory during tests on CI"
+    echo "  Setting default parallelism to 2 because we can run out of memory during tests on CI"
     echo " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
     echo
-    export AIRFLOW__CORE__PARALELLISM=2
+    export AIRFLOW__CORE__PARALLELISM=2
 fi
 
+cd "${AIRFLOW_SOURCES}"
+
 set +u
 # If we do not want to run tests, we simply drop into bash
-if [[ "${RUN_TESTS}" == "false" ]]; then
-    if [[ ${#ARGS} == 0 ]]; then
-        exec /bin/bash
-    else
-        exec /bin/bash -c "$(printf "%q " "${ARGS[@]}")"
-    fi
+if [[ "${RUN_TESTS:=false}" != "true" ]]; then
+    exec /bin/bash "${@}"
```

Comment:
Are you sure of it? Should not be added --c here?

Reply:
Yep. I am sure. We want to add -c manually if needed. If you do not add anything you are simply dropped into shell. This way you can also pass additional options to bash - not only -c.

See the examples in BREEZE.rst

Reply:
This way you can do "./breeze shell -- -c "ls" or "./docker run IMAGE bash -c "ls" which is exactly the way how Bash's docker is used.

Reply:
Right. I think it's not needed after recent HIVE mocking. Let's see what our tests have to say about it.